### ISSUE TITLE
As a developer, I need to know when I've introduced code that significantly reduced the performance of cellxgene.

### ISSUE THREAD

**Issue by maniarathi** [chanzuckerberg/single-cell#49]
**Story**: As a developer, I need to know when I've introduced code that significantly reduced the performance of cellxgene. A "selling" point of cellxgene is that scientists can interact with their datasets at interactive speed. As a developer, I need to know when I have detracted from that experience so that I don't spend time later down the line debugging the performance issues which are sometimes tricky to find.
**Need**: As a scientists I want to continue having interactive speed of cellxgene even as my datasets get larger to up to a few million cells.

**TL;DR**: Introduce a performance test suite to cellxgene.

**Comment by maniarathi** [comment]
Closing since all issues have been addressed.
